packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.service.js

```diff
@@ -160,22 +160,59 @@ export default class DedicatedServerInterfacesService {
   }
 
   setPrivateAggregation(serverName, name, interfacesToGroup) {
-    return this.Ola.v6().group(
-      { serverName },
-      {
+    return this.$http
+      .post(`/dedicated/server/${serverName}/ola/aggregation`, {
         name,
         virtualNetworkInterfaces: map(interfacesToGroup, 'id'),
-      },
-    ).$promise;
+      })
+      .then((task) => this.waitForTask(serverName, task.data.taskId));
   }
 
   setDefaultInterfaces(serverName, interfaceToUngroup) {
-    return this.Ola.v6().ungroup(
-      { serverName },
-      {
+    return this.$http
+      .post(`/dedicated/server/${serverName}/ola/reset`, {
         virtualNetworkInterface: interfaceToUngroup.id,
+      })
+      .then((task) => this.waitForTask(serverName, task.data.taskId));
+  }
+
+  waitForTask(serverName, taskId) {
+    return this.Poller.poll(
+      `/dedicated/server/${serverName}/task/${taskId}`,
+      {},
+      {
+        namespace: 'dedicated.server.interfaces.ola.aggregation',
+        method: 'get',
+        successRule: {
+          status: 'done',
+        },
+        errorRule: (task) => {
+          return task.status === 'ovhError' || task.status === 'customerError';
+        },
       },
-    ).$promise;
+    ).then(
+      () => true,
+      (error) => {
```

Comment:
Not possible to use a `catch` ?

Reply:
well this code is copy/pasted from https://github.com/ovh/manager/pull/3402
i'll update it when @radireddy 's PR will be merged